src/header-generator.js

```diff
@@ -0,0 +1,271 @@
+const fs = require('fs');
+const path = require('path');
+const { BayesianNetwork } = require('bayesian-network');
+const { default: ow } = require('ow');
+
+const headerNetworkDefinitionPath = path.join(__dirname, './headerNetworkDefinition.json');
+const inputNetworkDefinitionPath = path.join(__dirname, './inputNetworkDefinition.json');
+const browserHelperFilePath = path.join(__dirname, './browserHelperFile.json');
+
+const browserHttpNodeName = '*BROWSER_HTTP';
+const operatingSystemNodeName = '*OPERATING_SYSTEM';
+const deviceNodeName = '*DEVICE';
+const missingValueDatasetToken = '*MISSING_VALUE*';
+
+const http2SecFetchAttributes = {
+    mode: 'sec-fetch-mode',
+    dest: 'sec-fetch-dest',
+    site: 'sec-fetch-site',
+    user: 'sec-fetch-user',
+};
+
+const http1SecFetchAttributes = {
+    mode: 'Sec-Fetch-Mode',
+    dest: 'Sec-Fetch-Dest',
+    site: 'Sec-Fetch-Site',
+    user: 'Sec-Fetch-User',
+};
+
+function getRandomInteger(minimum, maximum) {
```

Comment:
Maybe we can add some simple jsdoc even for our internal functions? It is super helpful for IDE completions and hints.
I think the param typing should do.

Reply:
I generally don't like using comments for these short helper functions, since they should be self-explanatory. But will add the param typing for prepareHttpBrowserObject and prepareBrowserObject, since it might not be clear what the argument is supposed to be.

Reply:
Ok, make sense 🆗